astropy/coordinates/sky_coordinate.py

```diff
@@ -1163,6 +1163,40 @@ def spherical_offsets_to(self, tocoord):
         dlat = acoord.spherical.lat.view(Angle)
         return dlon, dlat
 
+    def spherical_offsets_by(self, d_lon, d_lat):
+        """
+        Computes the coordinates at the specified angular offsets.
```

Comment:
```suggestion
        Computes the coordinate that is a specified pair of angular offsets
        from this coordinate.
```
I found the original wording confusing because of the from/to ambiguity.